src/sql/src/plan.rs

```diff
@@ -598,6 +598,7 @@ pub struct SubscribePlan {
     pub up_to: Option<MirScalarExpr>,
     pub copy_to: Option<CopyFormat>,
     pub emit_progress: bool,
+    pub key_indices: Option<Vec<usize>>,
```

Comment:
```suggestion
    pub key_indices: Vec<usize>,
```

Is usually what we do here, and if on empty or not.

Reply:
I added another output type. I wasn't sure whether to do `output: Option<SubscribeOutput>` or `output: SubscribeOutput` and add a None enum to SubscribeOutput